tests/Api/Category/UpsertListCategoryIntegration.php

```diff
@@ -47,6 +47,39 @@ public function testUpsertListSuccessful()
         ], $responseLines[2]);
     }
 
+    public function testUpsertListFromStreamSuccessful()
+    {
+        $resources = fopen('php://memory', 'w+');
+        fwrite($resources,
+<<<JSON
+{"code":"sandals","parent":"winter_collection","labels":{"en_US":"Sandals","fr_FR":"Sandales"}}
+{"code":"booties","parent":"summer_collection","labels":{"en_US":"Booties","fr_FR":"Bottines"}}
+JSON
+        );
+        rewind($resources);
+
+        $streamedResources = $this->getStreamFactory()->createStream($resources);
+        $api = $this->createClient()->getCategoryApi();
+        $response = $api->upsertList($streamedResources);
+
+        $this->assertInstanceOf('\Iterator', $response);
+
+        $responseLines = iterator_to_array($response);
+        $this->assertCount(2, $responseLines);
+
+        $this->assertSame([
```

Comment:
you can transform both `assertSame` into one, and with that `asserCount` is not needed